data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,129 @@ mission "Coalition: Outskirts Gauger Ready"
+			choice
+				`	"I'm interested. Wait here, I'll go buy myself a copy."`
+					goto "buy her book"
+				`	"Cool. How has life been lately as an author?"`
+			label "life as an author sucks"
+			`	"Bad," says Ann. "I've never had a bestseller or even a book that surpasses 10,000 copies sold."`
+			choice
+				`	(Tell her about the Coalition.)`
+					goto "tell her about the coalition"
+				`	"Well, that's gotta hurt. I hope you find success in your life one day.`
+					decline
+			label "buy her book"
+			action
+				payment -60
+			`	You enter the bookstore, ask the employee for one of Ann's books, and purchase it for 60 credits.`
+			choice
+				`	(Now go to Ann and get it signed.)`
+					goto "get it signed"
+				`	(Leave.)`
+					decline
+			label "get it signed"
+			`	You move to the table where Ann is sitting. She looks visibly happy as she grabs your book, takes out a pen, and signs it. "Thanks for buying one of my books. I don't get much sales and people like you mean the world to me."`
+			choice
+				`	"I appreciate it. How has life as an author been though?"`
+					goto "life as an author sucks"
+				`	"Thanks for signing my copy. I hope you find success in your life. Goodbye."`
+					decline
+			label "tell her about the coalition"
+			`	You check your surroundings, looking around to ensure you are not being listened to. Once you confirm that your conversation will be private, you begin speaking.`
+			`	"You know, I know of a place called the Coalition. They're a group of alien races. Maybe you could go there and find inspiration for your next novel, or sell your books there? I bet they'd love to see your writings!"`
+			`	"Wait, really?" asks Ann.`
+			`	"Yes", you say.`
+			`	"But they wouldn't speak English. How are they going to read my books?"`
+			choice
+				`	"They have a translation device."`
+				`	"I'm sure they can learn the language."`
+			`	"That's reassuring... but, how am I even going to get there?"`
+			choice
+				`	"I'll take you there."`
+			`	"Do I have to pay anything? As a struggling author, I really can't pay you."`
+			choice
+				`	"It's okay. I'm willing to make an exception for you."`
+			`	"Thank you so much! Just wait for a while, I need to finish my book signing event. It should be roughly 45 minutes before it's over, and then we can go to whatever this Coalition thing is," says Ann.`
+			`	You reply. "Good luck. By the way, we're going to <destination> in Coalition space.`
+			choice
+				`	(Wait for Ann to finish her event.)`
+			`	After approximately an hour of waiting, Ann appears outside of your ship, knocking on the hatch. You open the hatch to let her inside, as you lead her to her bunk. You prepare to leave the spaceport.`
+				accept
+	on complete
+		conversation
+			`	You leave the ship with Ann following you. The moment you are out of your ship, three Kimeks surround you and Ann. The Kimek in the middle speaks, and then brief silence. The silence is broken as the yellow box speaks.`
```

Comment:
```suggestion
			`	You leave the ship with Ann following you. The moment you are out of your ship, three Kimek surround you and Ann. The Kimek in the middle speaks, and then brief silence. The silence is broken as the yellow box speaks.`
```
Kimek is both singular and plural.
Also are the boxes yellow? From what I gathered it's Heliarch circlets and armbands that are yellow. The boxes haven't been mentioned as to their colors from what I recall.